public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
If these files don't exist this file can probably be deleted to prevent 404 errors in the web console.